docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -30,7 +30,20 @@
       arg: system
       description: Stop Meshery
       usage: |
-        mesheryctl system start
+        mesheryctl system stop
```

Comment:
@Blakelist7 was this intended? Because the docs were required for `mesheryctl system config`

Reply:
Yeah, I thought this is was kinda incomplete so did a little part.

Reply:
I have removed this part, creating a new PR for this might be more feasible.